pkg/scheduler/framework/cycle_state.go

```diff
@@ -39,8 +39,6 @@ type StateKey string
 
 // CycleState provides a mechanism for plugins to store and retrieve arbitrary data.
 // StateData stored by one plugin can be read, altered, or deleted by another plugin.
-// CycleState does not provide any data protection, as all plugins are assumed to be
-// trusted.
```

Comment:
Why delete this? This sentence means, although rare, pluginB can mutate the data generated in pluginA.

Reply:
Oh sorry... I misunderstood its meaning before.

/close